docs/user_guides/projects/jobs/python_job.md

```diff
@@ -43,9 +43,20 @@ By default, the dialog will create a Spark job. To instead configure a Python jo
   </figure>
 </p>
 
-### Step 3: Set the script
+### Step 3: Set the job type
 
-Next step is to select the python script to run. You can either select `From project`, if the file was previously uploaded to Hopsworks, or `Upload new file` which lets you select a file from your local filesystem as demonstrated below.
+By default, the dialog will create a Spark job. To instead configure a Python job, select `PYTHON`.
+
+<p align="center">
+  <figure>
+    <img src="../../../../assets/images/guides/jobs/jobs_select_python.gif" alt="Select Python job type">
+    <figcaption>Select Python job type</figcaption>
+  </figure>
+</p>
+
+### Step 4: Set the script
+
+Next step is to select the python script to run. You can either select `From project`, if the file was previously uploaded to Hopsworks, or `Upload new file` which lets you select a file from your local filesystem as demonstrated below. By default, the job name is the same as the file name, but you can customize it as shown. 
```

Comment:
Next step is to select the python script to run. ->
Select the python script to run.